ODIN_II/SRC/parse_making_ast.cpp

```diff
@@ -285,7 +285,9 @@ static ast_node_t* resolve_symbol_node(ast_node_t* symbol_node) {
                 newNode = (ast_node_t*)current_scope->param_sc->data[sc_spot];
             }
 
-            if (newNode && newNode->types.variable.is_parameter == true) {
+            if (newNode && newNode->types.variable.is_localparam == true) {
```

Comment:
```
// we expect a node, otherwise there is something seriously worng here
oassert(newNode);
if ( newNode->types.variable.is_parameter || 
    newNode->types.variable.is_localparam) {
...
```

Reply:
Are the defparam stored in there too?